src/screens/Loading/index.js

```diff
@@ -25,7 +26,7 @@ const styles = StyleSheet.create({
   },
   content: {
     backgroundColor: 'rgba(0, 0, 0, 0.22)',
-    padding: 50,
-    borderRadius: 10
+    padding: dimen.normalizeDimen(50),
+    borderRadius: dimen.normalizeDimen(10)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Good job on normalizing the dimensions using `dimen.normalizeDimen()`. This will ensure that the UI looks consistent across different screen sizes and resolutions.
```diff
-    padding: 50,
-    borderRadius: 10
+    padding: dimen.normalizeDimen(50),
+    borderRadius: dimen.normalizeDimen(10)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->